mwsql-blogpost.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Remove "We can see that": this just makes the sentence longer and less direct; better to start with "The dump contains". </p><p>Remove "non-engineer audiences": just say "lowers the barrier for users with only basic Python knowledge".</p><p>I think the sentence "mwsql is meant to be a simple tool..." isn't really adding anything and could be removed.</p><p> </p><p>Don't use (down)loading, just say "downloading" or "loading and downloading" (but I'd prefer just "downloading", and I think it's clear enough even if the user might be "loading" the data somewhere other than "down" :-)).</p><p> </p><p>Change "Let's look at each of these features through a concrete example" to --&gt; "The rest of this tutorial will demonstrate each of these features through a concrete example." Then, add another section header to clearly signal the start of your tutorial / example content.  For example, a header like "Use mwsql with a Wiki data dump", followed by a sentence like "Let's use the Simple English Wikipedia database ['simplewiki'] to explore the features of mwsql.  The latest dumps can be found here".</p><p> </p><p>In the "Downloading dump files" section:</p><p>Present tense verbs would make this more succinct and slightly easier to read:</p><blockquote>If you access the dump files from a WMF-hosted environment, <code>mwsql</code> recognizes this and simply creates a pointer to the public directory where the dump file is found. If you are working on your local machine, the <code>load</code> function first downloads the dump file to your current working directory, and then creates a pointer to it once the download is complete.    </blockquote><blockquote>By default, <code>mwsql's</code> <code>load</code> function assumes that you want the 'latest' version of the dumps, but you can add a specific date as an optional parameter. If you don't use 'latest' , the date format should be 'YYYYMMDD'. Now, let's see this in action.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/blancadesal/wm-tutorials-and-blogposts/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done</p>